applications/asset_tracker_v2/src/cloud/cloud_codec/cloud_codec.h

```diff
@@ -293,6 +310,27 @@ int cloud_codec_init(struct cloud_data_cfg *cfg, cloud_codec_evt_handler_t event
 int cloud_codec_encode_neighbor_cells(struct cloud_codec_data *output,
 				      struct cloud_data_neighbor_cells *neighbor_cells);
 
+#if defined(CONFIG_LOCATION_METHOD_WIFI)
+/**
+ * @brief Encode cloud codec Wi-Fi access points data.
+ *
+ * @note LwM2M builds: This function does not output a list of objects, unlike other
+ *		       functions in this API. The object references that are required to update
+ *		       Wi-Fi access points are kept internal in the LwM2M utils library.
+ *
+ * @param[out] output String buffer for encoding result.
+ * @param[in] wifi_access_points Wi-Fi access points.
+ *
+ * @retval 0 on success
```

Comment:
Remember to end the sentences with a full stop.

Reply:
Added to this function and a lot more dots to the entire file.